### PR TITLE
Add friendly hints to doc

### PR DESCRIPTION
**Pull request by lou-lan**
**What type of PR is this?**
/kind documentation

**Which issue(s) this PR fixes**:
Add friendly hints to doc

### CODE REVIEW

File: docs/setup/setup.md [hunk 1]
```diff
@@ -61,19 +61,22 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 ### Run Cloud
 
 #### Run as a binary
-
++ Before beginning, though, you should be install gcc. Check command:
```
Comment (m1093782566) [inline docs/setup/setup.md]: Nice catch but we should fix the grammar issue in this sentence :)

Comment (lou-lan) [reply docs/setup/setup.md]: Please check it again

File: docs/setup/setup.md [hunk 2]
```diff
@@ -62,18 +62,19 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 
 #### Run as a binary
 
++ First, you can run `gcc -verison` check the gcc for install
```
Comment (rohitsardesai83) [inline docs/setup/setup.md]: verison -> version

Comment (rohitsardesai83) [reply docs/setup/setup.md]: i think command is gcc --version

Comment (lou-lan) [reply docs/setup/setup.md]: I updated it again

File: docs/setup/setup.md [hunk 3]
```diff
@@ -62,18 +62,19 @@ The cert/key will be generated in the `/etc/kubeedge/ca` and `/etc/kubeedge/cert
 
 #### Run as a binary
 
++ First, you can run `gcc -version` check the gcc for install
```
Comment (m1093782566) [inline docs/setup/setup.md]: Firstly, make sure gcc is already installed on your host. You can verify it via:

```
gcc -version
```

**Comment by m1093782566** [comment]
@lou-lan Please squash the commits.

**Comment by lou-lan** [comment]
@m1093782566 I've squashed the commits.

**Pull request merged**